### ISSUE TITLE
Can some results be made available please?

### ISSUE THREAD

**Issue by Sumeha** [davidtellez/contrastive-predictive-coding-images#1]
What classification accuracy do you get on MNIST and what is the accuracy/val loss for the CPC task itself?
I have been trying to implement this in Pytorch, but am not getting good results.

**Comment by Laliten** [comment]
I also get not goot results,. After running train_cpc.py, the accuracy I got was not very high about 66%. Then I run train_classifier.py, the accuracy I got was 22%. I don't know if there are any errors here, because I didn't change anything, just run

**Comment by mengdi-li** [comment]
> I also get not goot results,. After running train_cpc.py, the accuracy I got was not very high about 66%. Then I run train_classifier.py, the accuracy I got was 22%. I don't know if there are any errors here, because I didn't change anything, just run

The same from my side. Trying to improve it ...
